File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		String[] strs = new String[3];
		
		strs[0] = s.nextLine();
		strs[1] = s.nextLine();
		strs[2] = s.nextLine();
		
		for(int i = 0; i < 3; i++) {
			if(!strs[i].equals("FizzBuzz") && !strs[i].equals("Fizz") && !strs[i].equals("Buzz")) {
				String answer = method(Integer.parseInt(strs[i]) + (3 - i));
				System.out.println(answer);
				break;
			}
		}
		
	}
	
	public static String method(int num) {
		if(num % 3 == 0 && num % 5 == 0) {
			return "FizzBuzz";
		} else if(num % 3 == 0) {
			return "Fizz";
		} else if(num % 5 == 0) {
			return "Buzz";
		} else {
			return String.valueOf(num);
		}
	}
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.kt
fun main() {
    val strs = Array<String>(3){i -> ""}
    strs[0] = readln();
    strs[1] = readln();
    strs[2] = readln();
    
    for(i in 0..2) {
        if(!strs[i].startsWith("F") && !strs[i].startsWith("B")) {
            val answer = method(strs[i].toInt() + (3 - i))
            println(answer)
            break
        }
    }
}

fun method(num: Int): String =
    when {
        num % 3 == 0 && num % 5 == 0 -> "FizzBuzz"
        num % 3 == 0 -> "Fizz"
        num % 5 == 0 -> "Buzz"
        else -> num.toString()
    }